hazelcast/src/main/java/com/hazelcast/instance/impl/DefaultNodeExtension.java

```diff
@@ -165,7 +165,6 @@ public DefaultNodeExtension(Node node) {
     }
 
     private void checkPersistenceAllowed() {
-        // todo: to be replaced with PersistenceConfig
         HotRestartPersistenceConfig hotRestartPersistenceConfig = node.getConfig().getHotRestartPersistenceConfig();
```

Comment:
Minor: this still remained on `HRConfig`. Doesn't matter really, shouldn't affect anything, but we can already do the replacement here.

Reply:
Thanks :) applied the change